src/frontend/components/TourGuide/TourGuide.scss

```diff
@@ -0,0 +1,77 @@
+.introjs-tooltip {
+  background: #222530;
+  color: var(--text-default);
+  border-radius: var(--space-xs);
+  box-shadow: 0px 4px 20px rgba(0, 0, 0, 0.25);
+  border: 1px solid var(--accent);
```

Comment:
You should use the stroke variables we have now.

Reply:
Which ones?

Reply:
You can check the ones named "stroke" on our hp-ui color variables